zerver/webhooks/front/view.py

```diff
@@ -39,27 +39,27 @@ def get_target_name(payload: WildValue) -> str:
 
 
 def get_inbound_message_body(payload: WildValue) -> str:
-    link, outbox, inbox, subject = get_message_data(payload)
+    link, outbox, inbox, topic = get_message_data(payload)
```

Comment:
Here as well as in some other places in this file, the word "subject" shouldn't be changed with "topic". As here "subject" means the subject of the message and that shouldn't be replaced with topic.